packages/common/src/context/comments/commentsContext.tsx

```diff
@@ -76,6 +76,7 @@ export const CommentSectionProvider = (
   const [currentSort, setCurrentSort] = useState<TrackCommentsSortMethodEnum>(
     TrackCommentsSortMethodEnum.Top
   )
+  const [isMutating, setIsMutating] = useState(false)
```

Comment:
I don't love adding more state here; especially given the data is technically already in the store. Unfortunately since the data is tucked away behind hashed aquery args, the only other way to select that data would be to have the args for each mutation and use the args to find the data in a-query.
This is a much simpler approach albeit more react lifecycles